src/Units/Weapons/PipeGun.java

```diff
@@ -0,0 +1,11 @@
+package Units.Weapons;
+
+public class PipeGun extends WeaponModel
+{
+
+  public PipeGun()
+  {
+    super(WeaponType.PIPEGUN, 9, 2, 5);
```

Comment:
Instead of all these magic numbers, it's easier for future maintainers if you just declare some class-level variables, and pass those into `super`. That way I know what '9' means without going spelunking.

Reply:
Hmm. That does sound nice, but it's also a lot of typing I don't much feel like doing.